homeworks/nemethmate/8_streamlit/app.py

```diff
@@ -0,0 +1,91 @@
+import streamlit as st
+import requests
+import pandas as pd
+from datetime import datetime
+import sqlite3
+import toml
+
+# Inicializáljuk az adatbázist
+def init_db():
+    conn = sqlite3.connect('weather_logs.db')
+    c = conn.cursor()
+    c.execute('''
+        CREATE TABLE IF NOT EXISTS logs (
+            city TEXT,
+            temperature REAL,
+            humidity REAL,
+            wind_speed REAL,
+            timestamp DATETIME
+        )
+    ''')
+    conn.commit()
+    conn.close()
+
+# Adatok logolása az adatbázisba
+def log_weather(city, temperature, humidity, wind_speed):
+    conn = sqlite3.connect('weather_logs.db')
+    c = conn.cursor()
+    c.execute('INSERT INTO logs VALUES (?, ?, ?, ?, ?)', 
+              (city, temperature, humidity, wind_speed, datetime.now()))
+    conn.commit()
+    conn.close()
+
+# API kulcs betöltése a teljes elérési útvonal megadásával
+secrets_path = r"C:\Users\veszperenyine\Documents\GitHub\python_hu\homeworks\nemethmate\8_streamlit\secrets.toml"
```

Comment:
ezt a streamlit magától megcsinálja, ha van egy .streamlit mappa a repo-ban, és abban van egy secrets.toml.
ezután úgy tudsz hivatkozni a secret-re ahogy az órán is tanultuk.